### PR TITLE
test(robot-server): magneticModuleV1 /commands acceptance tests

### PR DESCRIPTION
**Pull request by y3rsh**
# Overview

- All acceptance tests for magneticModuleV1 at /commands

# Risk assessment
None, test only

### CODE REVIEW

File: robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      json:
+        data:
+          commandType: magneticModule/engageMagnet
+          status: failed
+          params:
+            moduleId: '{magneticModuleV1_id}'
+            engageHeight: 21
+          error:
+            errorType: EngageHeightOutOfRangeError
+            detail: 'Invalid engage height for magneticModuleV1: 47.0. Must be 0 - 45.'
```
Comment (y3rsh) [inline robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml:77]: So this error doesn't match the units.  I know @SyntaxColoring knows exactly why the error looks like this.  Should we leave the test like this?

Comment (SyntaxColoring) [reply robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml:77]: Yeah, this is a known problem. I'm sure it's solvable, but I don't have an easy solution for it. So I guess let's ticket it and leave the test as-is for now?

File: robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
+test_name: MagDeck standalone commands
```
Comment (SyntaxColoring) [inline robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml]: This is very pedantic and you should feel free to ignore it, but "magdeck" is a legacy identifier that we've been slowly moving away from for a while. I would spell this out as "Magnetic Module" to match the product branding and the HTTP API's identifiers like `magneticModuleV1`.

**Review by SyntaxColoring — APPROVED** [review]

**Comment by SyntaxColoring** [comment]
Looks good to me aside for the fact that the `waitUntilComplete` in this test and one other appears to be timing out for some reason. I guess a legit server bug?

File: robot-server/tests/integration/fixtures.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import requests
+from box import Box
```
Comment (mcous) [inline robot-server/tests/integration/fixtures.py:2]: Should we add this to `robot-server/Pipfile` + lockfile as a top-level dev dep? I see that it's a Tavern recommendation / dependency

**Review by mcous — APPROVED** [review]

File: robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml (hunk 2 above)
Comment (y3rsh) [reply robot-server/tests/integration/http_api/commands/test_mag_commands.tavern.yaml]: Changed 😊

File: robot-server/tests/integration/fixtures.py (hunk 3 above)
Comment (y3rsh) [reply robot-server/tests/integration/fixtures.py:2]: Yes I will add it.  Actually meant to write a comment asking, thanks for catching.  I think you must return Box objects from external function calls if you want access to the data but I did not dig in, just followed the docs on it.

**Pull request merged**